Update README file with simple demo and instruction (#27)
osscar-org/widget-bandsplot: README.md

Usage
-----

> ### 1. Plot both bandstucture and density of states (DOS) side by side

Reviewer: I'm pretty sure that this should be "band structure", compare: https://en.wikipedia.org/wiki/Electronic_band_structure
```suggestion
### 1. Plot both the band structure and the density of states (DOS) side by side
```

```python
w = BandsPlotWidget(bands=[banddata1, banddata2], dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -10.0, "ymax": 10.0})
display(w)
```

In order to plot the bandstructure and density of states fiugres, one needs

> to provide band data and DOS data as json files. The examples of the input

Author: Ok, I will do it.

> to provide band data and DOS data as json files. The examples of the input

Author: I agree that bands data is better.

> json files are given in the `test/data` folder. The json files for the

Reviewer: ```suggestion
JSON-files are provided in the `test/data` folder. The JSON-files for the
```

                "borderColor": "#41e2b3",
                "backgroundColor": "#51258b",
                "lineStyle": "dash",
            },
            {
                "label": "Co (s↑)",
                "x": [0.0, 0.1, 0.2],
                "y": [1.2, 3.2, 0.0],
                "lineStyle": "solid",
                "borderColor": "#43ee8b",
                "backgroundColor": "#59595c",
            },
            {
                "label": "Co (s↓)",
                "x": [0.0, 0.1, 0.2],
                "y": [1.2, 3.2, 0.0],
                "lineStyle": "solid",
                "borderColor": "#403bae",
                "backgroundColor": "#a16c5e",
            },
        ],
    }
```

### 2. Plot only the bandstructure

```python
w = BandsPlotWidget(bands=[banddata1, banddata2], plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -10.0, "ymax": 10.0})
display(w)
```

### 3. Plot only the density of states (DOS)

```python
w = BandsPlotWidget(dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -10.0, "ymax": 10.0})
display(w)
```

When only plotting the density of states, the figure will be shown in
horizontal format.

> For developer

Reviewer: ```suggestion
For developers
```

Usage
-----

### 1. Plot both the band structure and the density of states (DOS) side by side

```python
w = BandsPlotWidget(bands=[banddata1, banddata2], dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -10.0, "ymax": 10.0})
display(w)
```

In order to plot the bandstructure and density of states fiugres, one needs
to provide bands data and DOS data as JSON-files. The examples of the input

> JSON-files are provided in the `test/data` folder. The JSON-files for the

Reviewer 2: ```suggestion
JSON-files are provided in the `examples/data` folder. The JSON-files for the
```

bandstructure can be exported with the [AiiDA command line interface (CLI) `verdi`](https://aiida.readthedocs.io/projects/aiida-core/en/latest/reference/command_line.html#reference-command-line) as demonstrated in
the code below:

```bash

> verdi data band export PK --format=json

Reviewer 2: ```suggestion
verdi data band export <PK> --format=json
```

```

One can plot several bandstructure input files together with the

> "widget-bandsplot". However, one can only give one input file

Reviewer 2: ```suggestion
widget. However, one can only give one input file
```

display(w)
```

> In order to plot the band structure and density of states fiugres, one needs

Reviewer: Typo, but I'd suggest to drop the word entirely.
```suggestion
In order to plot the band structure and density of states, one needs
```

                "lineStyle": "dash",
            },
            {

>                 "label": "Co (s↑)",

Reviewer 2: Hi @dou-du, I suggest you open a separated PR for this specifically.

Author: Ok, I will open a separated PR for Giovanni's suggestion. However, since it is just a label here, I will not change it back.
